ansible/roles/screenly/files/screenly.conf

```diff
@@ -33,7 +33,7 @@ debug_logging = False
 ; Only display pages with proper SSL certificates
 verify_ssl = True
 
-; Run Resin wifi connect if there is no connection
+; Run Balena WiFi Connect if there is no connection
```

Comment:
Please add a task to resurrect the Balena WiFi Connect feature so that we can once again get WiFi working on the devices.